libafl/src/fuzzer/mod.rs

```diff
@@ -391,6 +401,7 @@ where
                     .append_metadata(state, observers, &mut testcase)?;
                 let idx = state.corpus_mut().add(testcase)?;
                 self.scheduler_mut().on_add(state, idx)?;
+                *state.pending_mut() += 1;
```

Comment:
when do you decrease this?

Reply:
i think this is more involved. you have to change scheduler to insert the code to check if it's the 1st time to mutate a corpus entry.

if you are unsure you can just leave `pending` to 0. we'll take care later